### PR TITLE
Implement a basic Board renderer

### PR DESCRIPTION
**Pull request by mbalc**

### CODE REVIEW

File: src/board.hpp [hunk 1]
```diff
@@ -3,13 +3,15 @@
...
+#include <array>
+#include <vector>
 
 namespace TicTacToe {
     class Board {
     public:
         std::optional <Symbol> fieldContent(Coords coords);
 
-        std::vector<Coords[3]> winningSymbolLines();
+        std::vector<std::array<Coords, BOARD_WIDTH>> winningSymbolLines();
```
Comment (mbalc) [inline src/board.hpp]: @olcia303 lmk if these array changes are up to date with what you've got pending for commit

File: src/board_renderer.cpp [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+            output[i] = board.fieldContent(fieldCoords);
+        }
+
+        return output;
+    }
+
+    std::string repeatStringWithSeparator(int repetitionCount, std::string repeatableString, char separator) {
+        std::vector <std::string> repetitions(repetitionCount, repeatableString);
+
+        // convert from char to string; TODO: change all chars to strings? avoid hack
```
Comment (mbalc) [inline src/board_renderer.cpp]: @mbalc TODO polish this code

File: src/board.hpp (hunk 1 above)
Comment (olcia303) [reply src/board.hpp]: Kinda, i added using BoardLine = std::array<Coords, BOARD_WIDTH> instead of putting it all in one line

**Review by olcia303 — APPROVED** [review]
It's alright

File: src/board.hpp (hunk 1 above)
Comment (mbalc) [reply src/board.hpp]: that's even better

File: src/board_renderer.cpp (hunk 2 above)
Comment (mbalc) [reply src/board_renderer.cpp]: resolved - 0c553b4

File: src/board.hpp (hunk 1 above)
Comment (mbalc) [reply src/board.hpp]: resolved - 8b202d6

**Pull request merged**